bambi/results.py

```diff
@@ -30,6 +31,10 @@ def plot(self):
     def summary(self):
         pass
 
+    @abstractmethod
+    def get_trace(self):
```

Comment:
I don't think we want to make this an abstract method, because not every Results class will have a `trace` attribute, only those based on MCMC sampling (e.g., right now the `PyMC3ADVIResults` class doesn't). If we want to be strict about this, we should probably add another class to the hierarchy that's called `MCMCModelResults` or something. But it seems like overkill for now, so maybe just cut this out and only leave it implemented on the `PyMCResults` class.